### DISCUSSION TITLE
KEDA Chart 2.6.1

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#244] [Announcements]
*Chart: v2.6.1 | App: v2.6.0*

## TL;DR
```shell
helm repo add keda https://kedacore.github.io/charts
helm install keda/keda
```

### Features

None.

### Fixes / Changes

- Provide parameters per container for resources & security context (https://github.com/kedacore/charts/pull/239 - Contributed by @AshutoshNirkhe 🎉)
- Fix indentation of `ServiceAccount` annotations (https://github.com/kedacore/charts/pull/236 - Contributed by @bmalinconico🎉)

### Breaking Changes

None.

### Removal

None.

## New Contributors
* @AshutoshNirkhe made their first contribution in https://github.com/kedacore/charts/pull/239
* @bmalinconico made their first contribution in https://github.com/kedacore/charts/pull/236

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.6.0...v2.6.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.6.1'>KEDA Chart 2.6.1</a>.</em>
